src/x86/rfxcodec_encode_dwt_shift_x86_sse2.asm

```diff
@@ -21,6 +22,29 @@
 
 %ifidn __OUTPUT_FORMAT__,elf
 section .note.GNU-stack noalloc noexec nowrite progbits
+%ifdef PIC
```

Comment:
Oh and, PIC (implemented like this) is an ELF-specific phenomenon anyway… other object formats may use different mechanisms.